src/expr/src/relation/func.rs

```diff
@@ -3516,4 +3605,100 @@ mod tests {
             assert_eq!(actual.unwrap(), expect);
         }
     }
+
+    /// This is not a real test, but a benchmark.
```

Comment:
Can you convert it to a real benchmark, like we do elsewhere? For example, there's a benchmark in `interchange`.